packages/ui/atoms/Icon/Icon.twig

```diff
@@ -4,10 +4,24 @@
  * Icon component
  *
  * @param string $name The file name of the icon.
+ * @param array $attr Custom attributes.
  * @param string[] $classes Additional classes.
  */
 #}
 
-<span class="icon {{ 'icon--%s'|format(name) }} {{ html_classes(classes|default('inline-block')) }}">
+{% set attributes = merge_html_attributes(
+  attr ?? null,
+  {
+    class: classes|default('inline-block'),
+  },
+  {
+    class: [
+      'icon',
+      'icon--%s'|format(name),
+    ],
+  }
```

Comment:
**issue:** Separate the default value from the `classes` variable to avoid a breaking change

The `classes` parameter will be overriden by the `attr` parameter if it defines a `class` key. To avoid potentially missing values defined in the `classes` parameter, we need to add it to the required attributes. 

```suggestion
  {
    class: 'inline-block',
  },
  {
    class: [
      'icon',
      'icon--%s'|format(name),
      classes
    ],
  }
```